docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,120 @@
+
+.. image:: /assets/images/docs/add-workflow-button.png
+   :width: 40 %
+
+.. image:: /assets/images/docs/register-workflow-github-apps.png
+   :width: 40 %
+
+.. image:: /assets/images/docs/gh-app-reg-1.png
+   :width: 40 %
+
+Creating a .dockstore.yml File
+-------------------------------
+
+Once the GitHub app is installed on the correct repo, the next step is to create a ``/.dockstore.yml`` file. We'll cover a very straigtforward example
+first, but depending on how you configured the workflow during registration and whether your GitHub repository houses multiple workflows published on Dockstore,
+there will be additional steps to writing your ``/.dockstore.yml`` file.
+
+Let's say we have the following CWL workflow registered on Dockstore that came from this `repository <https://github.com/NatalieEO/ghapps-single-workflow>`__ and you would like to convert the master branch.
+
+.. figure:: /assets/images/docs/single-workflow-to-migrate.png
+   :alt: Workflow to Migrate
+
+As noted in our other documentation, create a ``/.dockstore.yml`` file in the root directory of the master branch in your repository. The file should look like the following
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test/dockstore.cwl.json
+
+The information above was filled out using the following:
+
+- ``subclass`` is taken from the ``Descriptor Type``
+- ``primaryDescriptorPath`` is from ``Workflow Path``
+- ``testParameterFiles`` is from ``Test File Path``
+
+During the original registration for your workflow, you may have filled out the ``Workflow Name`` field shown in the picture below.
+
+.. figure:: /assets/images/docs/workflow-name-field.png
+   :alt: Workflow to Migrate
+   :width: 60 %
+
+This field is required when you want to register multiple workflows from the same repo, but you may have filled it out for other reasons. To check if the workflow
+you want to migrate has a workflow name set, select the workflow and look at the title on top as shown in the picture below.
+
+The title consists of:
+``<sourceControl>/<organization name>/<repository name>/<optional workflow name>:<version name>``
+
+If you see a workflow name inserted, you must include the name field in your ``/.dockstore.yml``.
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test/dockstore.cwl.json
+        name: optional-name
+
+If you have multiple workflows stemming from the same repo, a single ``/.dockstore.yml`` can be used to convert them. Again, you need to check for the Workflow Name field being set
+because in this case, all but one workflow must have this field set (e.g. if you have three workflows from the same repo registered on Dockstore, at least two of them will have the Workflow Name set).
```

Comment:
Perhaps we should just say multi workflow repositories need to use a name for everything, might be less confusing.

Reply:
reworded